minishift/README

```diff
@@ -0,0 +1,103 @@
+## Start running dependent services on OpenShift
+
+These instructions will help you run your services on OpenShift using MiniShift.
+
+### Prerequisites
+
+
+[Kedge](kedgeproject.org)
+
+[MiniShift](https://docs.openshift.org/latest/minishift/getting-started/installing.html)
+
+[Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+
+### Installation (Linux)
+
+##### Install Kedge
+
+Following steps will download and install Kedge on your machine and put it in your $PATH. For more detailed information please visit kedgeproject.org
+
+```
+curl -L https://github.com/kedgeproject/kedge/releases/download/v0.5.1/kedge-linux-amd64 -o kedge
+```
+
+Verify installation by running following command, you should get version number.
+
+```
+kedge version
+```
+
+##### Install Minishift
+
+Make sure you have all prerequisites installed. Please check the list [here](https://docs.openshift.org/latest/minishift/getting-started/installing.html#install-prerequisites)
+
+Download and put `minishift` in your $PATH by following steps [here](https://docs.openshift.org/latest/minishift/getting-started/installing.html#manually)
+
+Verify installation by running following command, you should get version number.
+```
+minishift version
+```
+
+
+##### Install Kubectl
+we will use `curl` to download Kubectl, please execute following commands
+```
+
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+
+chmod +x ./kubectl
+
+sudo mv ./kubectl /usr/local/bin/kubectl
+```
+
```

Comment:
You should point to https://kubernetes.io/docs/tasks/tools/install-kubectl/ , similar to how you pointed to Kedge and Minishift documentation in this documentation. I would suggest you to only point to the upstream  docs as it is hard to maintain same documentation in two different places. It can become a maintenance nightmare.

Reply:
thanks, @LalatenduMohanty , yes I could see that thing.
I will update it. Just wanted the reader to stay on the page as steps are pretty simple and straightforward.

Reply:
done in https://github.com/fabric8-services/fabric8-wit/pull/1781/commits/b612a2bda36a17a0c8237d496f78e9b13e340f2c